### PR TITLE
feat: update blocto version

### PR DESCRIPTION
**Pull request by sanyu1225**
### Description

update blocto sdk version

### Checklist
- [X] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [X] Check the box that allows repo maintainers to update this PR
- [X] Test locally to make sure this feature/fix works
- [X] Run `yarn check-all` to confirm there are not any associated errors
- [X] Confirm this PR passes Circle CI checks
- [X] Add or update relevant information in the documentation

### Docs Checklist
- [X] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [X] Add/update the appropriate package README (if applicable)
- [X] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [X] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [X] send transaction
- [X] switch chains
- [X] sign message
- [X] sign typed message
- [X] disconnect

#### Tests with demo app (SDK)
- [X] send transaction
- [X] switch chains
- [X] sign message
- [X] sign typed message
- [X] disconnect

### CODE REVIEW

File: packages/blocto/package.json [hunk 1]
```diff
@@ -66,6 +66,6 @@
   },
   "dependencies": {
     "@web3-onboard/common": "^2.3.1",
-    "@blocto/sdk": "^0.4.6"
+    "@blocto/sdk": "^0.9.1"
```
Comment (Adamj1232) [inline packages/blocto/package.json:69]: We will need an alpha version update on line 3 of this file ie `2.0.1-alpha.1`

File: packages/blocto/src/index.ts [hunk 2]
```diff
@@ -28,9 +28,7 @@ function BloctoWallet(): WalletInit {
               return item.id === (params && params[0] && params[0].chainId)
             })
             if (!chain) throw new Error('chain must be set before switching')
-            const providerRpcurl =
-              bloctoProvider.switchableNetwork[chain.id] &&
-              bloctoProvider.switchableNetwork[chain.id].rpc_url
+            const providerRpcurl = bloctoProvider.rpc && bloctoProvider.rpc
```
Comment (Adamj1232) [inline packages/blocto/src/index.ts:31]: Are both of these needed?

File: packages/core/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/core",
-  "version": "2.21.3-alpha.1",
+  "version": "2.21.3-alpha.2",
```
Comment (Adamj1232) [inline packages/core/package.json:3]: This version change isn't necessary

File: packages/solid/package.json [hunk 4]
```diff
@@ -63,7 +63,7 @@
   },
   "dependencies": {
     "@web3-onboard/common": "^2.3.3",
-    "@web3-onboard/core": "^2.21.3-alpha.1",
+    "@web3-onboard/core": "^2.21.3-alpha.2",
```
Comment (Adamj1232) [inline packages/solid/package.json:66]: This version change isn't necessary

File: packages/vue/package.json [hunk 5]
```diff
@@ -62,7 +62,7 @@
     "@vueuse/core": "^8.4.2",
     "@vueuse/rxjs": "^8.2.0",
     "@web3-onboard/common": "^2.3.3",
-    "@web3-onboard/core": "^2.21.3-alpha.1",
+    "@web3-onboard/core": "^2.21.3-alpha.2",
```
Comment (Adamj1232) [inline packages/vue/package.json:65]: This version change isn't necessary - please revert

Comment (sanyu1225) [reply packages/vue/package.json]: fixed in this commit [a065226](https://github.com/blocknative/web3-onboard/pull/2082/commits/a065226661afe3a67d91c13193171ba249611675) , thx

File: packages/solid/package.json (hunk 4 above)
Comment (sanyu1225) [reply packages/solid/package.json]: fixed in this commit [a065226](https://github.com/blocknative/web3-onboard/pull/2082/commits/a065226661afe3a67d91c13193171ba249611675) , thx

File: packages/react/package.json [hunk 6]
```diff
@@ -62,7 +62,7 @@
   },
   "dependencies": {
     "@web3-onboard/common": "^2.3.3",
-    "@web3-onboard/core": "^2.21.3-alpha.1",
+    "@web3-onboard/core": "^2.21.3-alpha.2",
```
[the remark this answers is not in the document]
Comment (sanyu1225) [reply packages/react/package.json]: fixed in this commit [a065226](https://github.com/blocknative/web3-onboard/pull/2082/commits/a065226661afe3a67d91c13193171ba249611675) , thx

File: packages/blocto/src/index.ts [hunk 7]
```diff
@@ -28,9 +28,9 @@ function BloctoWallet(): WalletInit {
               return item.id === (params && params[0] && params[0].chainId)
             })
             if (!chain) throw new Error('chain must be set before switching')
-            const providerRpcurl =
-              bloctoProvider.switchableNetwork[chain.id] &&
-              bloctoProvider.switchableNetwork[chain.id].rpc_url
+            const providerRpcurl = bloctoProvider.rpc
+              ? bloctoProvider.rpc
+              : undefined
```
Comment (Adamj1232) [inline packages/blocto/src/index.ts:33]:
```suggestion
            const providerRpcurl = bloctoProvider?.rpc
```

File: packages/demo/package.json [hunk 8]
```diff
@@ -33,7 +33,7 @@
     "@web3-onboard/capsule": "2.0.1",
     "@web3-onboard/cede-store": "^2.2.0",
     "@web3-onboard/coinbase": "^2.2.6",
-    "@web3-onboard/core": "^2.21.3-alpha.1",
+    "@web3-onboard/core": "^2.21.3-alpha.2",
```
[the remark this answers is not in the document]
Comment (sanyu1225) [reply packages/demo/package.json]: fixed in this commit [a065226](https://github.com/blocknative/web3-onboard/pull/2082/commits/a065226661afe3a67d91c13193171ba249611675) , thx

File: packages/core/package.json (hunk 3 above)
Comment (sanyu1225) [reply packages/core/package.json]: fixed in this commit [a065226](https://github.com/blocknative/web3-onboard/pull/2082/commits/a065226661afe3a67d91c13193171ba249611675) , thx

File: packages/blocto/src/index.ts (hunk 2 above)
Comment (sanyu1225) [reply packages/blocto/src/index.ts]: fixed in this commit [a065226](https://github.com/blocknative/web3-onboard/pull/2082/commits/a065226661afe3a67d91c13193171ba249611675) , thx

File: packages/blocto/src/index.ts (hunk 7 above)
Comment (Adamj1232) [reply packages/blocto/src/index.ts:33]: Can we also fix the casing on this var to `providerRpcUrl`?

Comment (sanyu1225) [reply packages/blocto/src/index.ts]: sure, fixed in [fa5edc6](https://github.com/blocknative/web3-onboard/pull/2082/commits/fa5edc67f6cc3a7ac1290545e2c23889ceb30572) and  aedd7d5

**Review by Adamj1232 — APPROVED** [review]
Nice work! LGTM!!

**Pull request merged**